packages/nextjs/src/utils/instrumentServer.ts

```diff
@@ -111,18 +110,6 @@ function makeWrappedHandlerGetter(origHandlerGetter: HandlerGetter): WrappedHand
   // Otherwise, it's just a pass-through to the original method.
   const wrappedHandlerGetter = async function(this: NextServer): Promise<ReqHandler> {
     if (!sdkSetupComplete) {
-      try {
-        // `SENTRY_SERVER_INIT_PATH` is set at build time, and points to a webpack-processed version of the user's
-        // `sentry.server.config.js`. Requiring it starts the SDK.
-        require(path.resolve(process.env.SENTRY_SERVER_INIT_PATH as string));
```

Comment:
Why remove this?

Reply:
It’s no longer required because of the changes in packages/nextjs/src/config/webpack.ts

Reply:
Yeah, that's the whole point of this change - stop manually requiring a file whose location we have to store in another file, because that's too brittle a system.